app/services/ucas_matches/resolve_on_ucas.rb

```diff
@@ -7,8 +7,10 @@ def initialize(ucas_match)
     end
 
     def call
+      automatic_withdrawal ||= ucas_match.ucas_withdrawal_requested?
```

Comment:
Could `automatic_withdrawal` ever have been initialised?

Reply:
I decided to do it this way to access the previous value of `action_taken` as it gets updated to `resolved_on_ucas` on line 12. 
On reflection, I should probably use `action_taken_was == 'ucas_withdrawal_requested'`